<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="c0f6c8b7-51d4-4fb0-b439-5a2689ed2cd6" name="800040443@4CJSF2X 2011-09-29 11:53:52" runUser="CHNCORP\800040443" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="d8a11410-e764-4fd1-a59f-afb07cd8c00b">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="800040443_4CJSF2X 2011-09-29 11_53_52" />
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2011-09-29T11:53:52.1834822+08:00" queuing="2011-09-29T11:53:52.8709030+08:00" start="2011-09-29T11:53:53.2458598+08:00" finish="2011-09-29T11:53:54.3551070+08:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="CreateTable" storage="e:\onote\unittest\bin\debug\unittest.dll" id="dd42e8bb-dcb6-51f1-dd26-cd74f6c669b8">
      <Execution timeOut="1800000" id="9edee3a6-a4c2-4a1c-b823-2543ea644723" />
      <TestMethod codeBase="E:/ONote/UnitTest/bin/Debug/UnitTest.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="UnitTest.UnitTest1, UnitTest, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="CreateTable" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="dd42e8bb-dcb6-51f1-dd26-cd74f6c669b8" executionId="9edee3a6-a4c2-4a1c-b823-2543ea644723" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="9edee3a6-a4c2-4a1c-b823-2543ea644723" testId="dd42e8bb-dcb6-51f1-dd26-cd74f6c669b8" testName="CreateTable" computerName="4CJSF2X" duration="00:00:00.8025029" startTime="2011-09-29T11:53:53.3395990+08:00" endTime="2011-09-29T11:53:54.2457446+08:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="9edee3a6-a4c2-4a1c-b823-2543ea644723">
      <Output>
        <ErrorInfo>
          <Message>Test method UnitTest.UnitTest1.CreateTable threw exception: 
NHibernate.Cfg.HibernateConfigException: An exception occurred during configuration of persistence layer. ---&gt; System.IO.FileNotFoundException: Could not find file 'E:\ONote\UnitTest\bin\Debug\hibernate.cfg.xml'.</Message>
          <StackTrace>    at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy, RequestCachePolicy cachePolicy)
   at System.Xml.XmlUrlResolver.GetEntity(Uri absoluteUri, String role, Type ofObjectToReturn)
   at System.Xml.XmlTextReaderImpl.OpenUrlDelegate(Object xmlResolver)
   at System.Threading.CompressedStack.runTryCode(Object userData)
   at System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   at System.Threading.CompressedStack.Run(CompressedStack compressedStack, ContextCallback callback, Object state)
   at System.Xml.XmlTextReaderImpl.OpenUrl()
   at System.Xml.XmlTextReaderImpl.Read()
   at System.Xml.XmlTextReader.Read()
   at System.Xml.XmlCharCheckingReader.Read()
   at System.Xml.XsdValidatingReader.Read()
   at System.Xml.XPath.XPathDocument.LoadFromReader(XmlReader reader, XmlSpace space)
   at System.Xml.XPath.XPathDocument..ctor(XmlReader reader, XmlSpace space)
   at NHibernate.Cfg.ConfigurationSchema.HibernateConfiguration..ctor(XmlReader hbConfigurationReader, Boolean fromAppSetting)
 --- End of inner exception stack trace ---
    at NHibernate.Cfg.ConfigurationSchema.HibernateConfiguration..ctor(XmlReader hbConfigurationReader, Boolean fromAppSetting)
   at NHibernate.Cfg.Configuration.Configure(XmlReader textReader)
   at NHibernate.Cfg.Configuration.Configure(String fileName, Boolean ignoreSessionFactoryConfig)
   at NHibernate.Cfg.Configuration.Configure()
   at UnitTest.UnitTest1.CreateTable() in E:\ONote\UnitTest\UnitTest1.cs:line 16
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>